core/src/com/unciv/logic/city/CityInfo.kt

```diff
@@ -696,7 +696,10 @@ class CityInfo {
 
         // The relinquish ownership MUST come before removing the city,
         // because it updates the city stats which assumes there is a capital, so if you remove the capital it crashes
-        getTiles().forEach { expansion.relinquishOwnership(it) }
+        for (tile in getTiles()) {
+            tile.removeCreatesOneImprovementMarker()
+            expansion.relinquishOwnership(tile)
+        }
```

Comment:
Would it make sense to put the `tile.removeCreatesOneImprovementMarker()` call in the `relinquishOwnership()` function?

Reply:
~~I wanted to have the "what's that 'improvement marker'" knowledge in one place.~~ Oh you mean the call not the function. Okay, yes, that would also solve the case where a citadel drop steals the marked tile, which I'm almost certain the current state of the PR would miss.